Mongo.Migration.Test/Migrations/Database/DatabaseMigrationRunner_when_migrating_up.cs

```diff
@@ -14,17 +15,17 @@ internal class DatabaseMigrationRunner_when_migrating_up : DatabaseIntegrationTe
         private IDatabaseMigrationRunner _runner;
 
         [SetUp]
-        public void SetUp()
+        public async Task SetUpAsync()
         {
-            base.OnSetUp(DocumentVersion.Empty());
+            await base.OnSetUpAsync(DocumentVersion.Empty());
 
             this._runner = this._components.Get<IDatabaseMigrationRunner>();
         }
```

Comment:
`[SetUp]` not exists in `xUnit` but you can use `IAsyncLifetime`

https://mderriey.com/2017/09/04/async-lifetime-with-xunit/

Reply:
Done!